src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private BetDao betDao;
+    private UserDao userDao;
+    private String command;
+    private Scanner scanner;
```

Comment:
```suggestion
    private BetDao betDao = new BetDaoImpl();
    private UserDao userDao = new UserDaoImpl();
```

remove `command` and `scanner` fields and create `BetDao` and `UserDao` that are implemented by `BetDaoImpl` and  `UserDaoImpl` classes